Library/Homebrew/linkage_checker.rb

```diff
@@ -233,6 +241,24 @@ def harmless_broken_link?(dylib)
     ].include?(dylib)
   end
 
+  def check_packages
```

Comment:
Feels like this is somewhat stretch the definition of `linkage_checker`. Could this be a post-install audit instead?

Reply:
I think so. There was differing opinions when I asked a few weeks back so wasn't entirely sure.